### ISSUE TITLE
Get Embedding layer working in Keras

### ISSUE THREAD

**Issue by divilian** [TromboneDavies/PolarOps#50]
And see how well the auto-pre-train-your-own-word-embedding technique works.

**Comment by divilian** [comment]
Verdict: it kinda sucks. :(   (no appreciable difference in accuracy, at least with our tiny 158-thread data set.)
